src/battle_game/core/game_core.h

```diff
@@ -203,6 +206,7 @@ class GameCore {
   std::vector<std::function<uint32_t(uint32_t)>>
       primary_unit_allocation_functions_;
   std::vector<std::string> selectable_unit_list_;
+  std::vector<bool> selectable_unit_list_skill_;
```

Comment:
What is the purpose of this list? Please add some explanation in comments

Reply:
In case someone do not want their skills to be displayed.